install/tools/man/ipa-migrate.1

```diff
@@ -0,0 +1,83 @@
+.\"
+.\" Copyright (C) 2023  FreeIPA Contributors see COPYING for license
+.\"
+.TH "ipa-migrate" "1" "Aug 30 2023" "IPA" "IPA Manual Pages"
+.SH "NAME"
+ipa\-migrate \- Migrate an IPA server from one machine to another
+.SH "SYNOPSIS"
+ipa\-migrate
+.SH "DESCRIPTION"
+
+Use the \fIipa-migrate\fR command to migrate one
+IPA server to an existing local IPA server installation.
+
+Migrate IPA schema, configuration, and database to a local IPA server.  This
+migration can be done online, where the tool will query the remote server. Or,
+offline where LDIF files can be provided.  You can mix and match online and
+offline. So for example you could migrate the schema and configuration online,
+and then use an exported LDIF file for the database migration portion (this
+might be more useful for very large databases as you don't need to worry about
+network interruptions)
+
+.SH "COMMANDS"
+.TP
+\fB\-v\fR, \fB\-\-verbose\fR
+Use verbose output while running hte migration tool.
+.TP
+\fB\-\-hostname=HOSTNAME\fR
+The host name of the remote IPA server that is being migrated from.
+.TP
+\fB\-\-bind\-dn=BIND_DN\fR
+The Bind DN (Distinguished Name) or an LDAP entry to bind to the remote IPA server with.
+Typically this is "cn=directory manager".
```

Comment:
I think it should be mentioned that the bind user must be able to read the userPassword attribute for passwords to be migrated.